src/scalarstats.jl

```diff
@@ -536,12 +536,12 @@ If `normalize` is set to `true`, the MAD is multiplied by
 `1 / quantile(Normal(), 3/4) ≈ 1.4826`, in order to obtain a consistent estimator
 of the standard deviation under the assumption that the data is normally distributed.
 """
-function mad(x; center=nothing, normalize::Union{Bool, Nothing}=nothing, constant=nothing)
-    mad!(Base.copymutable(x); center=center, normalize=normalize, constant=constant)
+function mad(x; center=nothing, normalize::Union{Bool, Nothing}=nothing)
```

Comment:
Since this is a breaking change, I think we should remove support for `normalize::Nothing` since seemingly it's only purpose is to print a deprecation warning.
```suggestion
function mad(x; center=nothing, normalize::Bool=false)
```